assignments/array-methods.js

```diff
@@ -55,29 +55,58 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+
 let fullName = [];
+runners.forEach(function(elem) {
+    fullName.push(`${elem.first_name} ${elem.last_name}`)
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+
+let allCaps = runners.map(function (elem) {
+return elem.first_name.toUpperCase();
```

Comment:
This line should be indented, as it is inside of our callback function, but otherwise it is perfect